main/FatesHistoryInterfaceMod.F90

```diff
@@ -5226,13 +5405,81 @@ subroutine define_history_vars(this, initialize_variables)
+!         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+!         ivar=ivar, initialize=initialize_variables, index = ih_fire_disturbance_rate_si )
+!
+!    call this%set_history_var(vname='DISTURBANCE_RATE_LOGGING', units='m2 m-2 d-1',                   &
+!         long='Disturbance rate from logging',  use_default='active',     &
+!         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+!         ivar=ivar, initialize=initialize_variables, index = ih_logging_disturbance_rate_si )
+!
+!    call this%set_history_var(vname='DISTURBANCE_RATE_TREEFALL', units='m2 m-2 d-1',                   &
+!         long='Disturbance rate from treefall',  use_default='active',     &
+!         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+!         ivar=ivar, initialize=initialize_variables, index = ih_fall_disturbance_rate_si )
+!
+!    call this%set_history_var(vname='DISTURBANCE_RATE_POTENTIAL', units='m2 m-2 d-1',                   &
+!         long='Potential (i.e., including unresolved) disturbance rate',  use_default='active',     &
+!         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+!         ivar=ivar, initialize=initialize_variables, index = ih_potential_disturbance_rate_si )
+
+    call this%set_history_var(vname='HARVEST_CARBON_FLUX', units='kg C m-2 d-1',                   &
+         long='Harvest carbon flux',  use_default='active',     &
+         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+         ivar=ivar, initialize=initialize_variables, index = ih_harvest_carbonflux_si )
+
+    call this%set_history_var(vname='HARVEST_DEBT', units='kg C',                   &
+         long='Accumulated carbon failed to be harvested',  use_default='active',     &
+         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+         ivar=ivar, initialize=initialize_variables, index = ih_harvest_debt_si )
+
+    call this%set_history_var(vname='HARVEST_DEBT_SEC', units='kg C',                   &
+         long='Accumulated carbon failed to be harvested from secondary patches',  use_default='active',     &
+         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=1,   &
+         ivar=ivar, initialize=initialize_variables, index = ih_harvest_debt_sec_si )
+
+    ! Canopy Resistance 
+
+    call this%set_history_var(vname='C_STOMATA', units='umol m-2 s-1',                   &
+         long='mean stomatal conductance', use_default='active',                   &
+         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=2,   &
+         ivar=ivar, initialize=initialize_variables, index = ih_c_stomata_si )
+
+    call this%set_history_var(vname='C_LBLAYER', units='umol m-2 s-1',                   &
+         long='mean leaf boundary layer conductance', use_default='active',                   &
+         avgflag='A', vtype=site_r8, hlms='CLM:ALM', upfreq=2,   &
+         ivar=ivar, initialize=initialize_variables, index = ih_c_lblayer_si )
+
```

Comment:
```suggestion
```

These are all redundant to other variables, I think all of this code got accidentally maintained through a git merge?

Reply:
Yes, I did a merge to the master branch before. Will remove them.